programs/splitwave/src/instructions/create_splitwave_id.rs

```diff
@@ -11,10 +11,11 @@ pub struct CreateSplitwaveId<'info> {
         seeds = [SEED_SPLITWAVE_ID],
         bump
     )]
-    splitwave_id: Account<'info, SplitwaveId>,
+    splitwave_id: Box<Account<'info, SplitwaveId>>,
 
     #[account(mut)]
     payer: Signer<'info>,
+    pub rent: Sysvar<'info, Rent>,
```

Comment:
Why is this "pub" only added to this place? Can we use the same coding style in codebase?

Reply:
fixed in next commit